data/remnant/remnant outfits.txt

```diff
@@ -276,6 +277,136 @@ outfit "EMP Torpedo Bay"
 		"missile strength" 80
 	description "Electromagnetic pulse weapons were developed by the Remnant during the early days of their colony, when they were living in fear that the Alphas would overrun human space and expand beyond it."
 
+
+
+outfit "Teciimach Canister"
+	plural "Teciimach Canisters"
+	category "Ammunition"
+	licenses
+		Remnant
+	cost 5500
+	thumbnail "outfit/emp torpedo"
+	"mass" 1
+	"teciimach canister capacity" -1
+	description "This canister contains everything required to assemble a Teciimach torpedo. These weapons emit strong electromagnetic pulses which interfere with a ship's generators and neutralize a portion of their energy."
+	description "Unlike the previous generation of 'EMP torpedoes', the Teciimach torpedoes synchronize themselves with their mothership in order to avoid impacting their allies."
+
+outfit "Teciimach Canister Rack"
+	plural "Teciimach Canister Racks"
+	category "Ammunition"
+	licenses
+		Remnant
+	cost 98000
+	thumbnail "outfit/emp rack"
+	"mass" 5
+	"outfit space" -9
+	"energy capacity" 500
+	"teciimach canister capacity" 9
+	ammo "Teciimach Canister"
+	description "The Teciimach Canister rack is used to store extra sets of components, both to expand the capacity of a ship, and on carriers for resupplying fighters."
+	description "	In addition to its ammunition capacity, the storage bay can also contain a small amount of charge, enough to let an otherwise battery-less ship assemble Teciimach torpedoes."
+
+outfit "Teciimach Bay"
+	plural "Teciimach Bays"
+	category "Secondary Weapons"
+	licenses
+		Remnant
+	cost 583000
+	thumbnail "outfit/teciimach bay"
+	"mass" 9
+	"outfit space" -13
+	"weapon capacity" -13
+	"gun ports" -1
+	"teciimach canister capacity" 9
+	weapon
+		sprite "projectile/emp torpedo"
+			"frame rate" 2
+			"no repeat"
+		sound "torpedo"
+		ammo "Teciimach Canister"
+		icon "icon/emp"
+		"fire effect" "emp torpedo fire" 5
+		"hit effect" "nuke explosion"
+		"hit effect" "emp ring" 30
+		"hit effect" "emp spark" 20
+		"die effect" "tiny explosion"
+		cluster
+		safe
+		"reload" 310
+		"firing energy" 400
+		"firing heat" 18
+		"inaccuracy" 5
+		"lifetime" 300
+		"velocity" 8
+		"acceleration" .7
+		"drag" .1
+		"turn" 2.5
+		"homing" 3
+		"infrared tracking" .8
+		"radar tracking" .9
+		"trigger radius" 50
+		"blast radius" 200
+		"shield damage" 900
+		"hull damage" 125
+		"disabled damage" 60
```

Comment:
Hence why it still does disabled damage instead of being zeroed out entirely. Also, my assumption is that a disabled ship does have physical problems. If the wire isn't even connected to the computer anymore because a chunk got blasted out of it earlier in the battle, then that wire isn't going to be conducting charge into the computer. I'm not saying that there's nothing left to affect, I'm saying that the pathways to concentrate charge at dangerous levels are going to be vastly reduced. If you have a ham radio that is powered on and hooked up to an antenna, it will take a lot more EMP damage than an equivalent radio that is unpowered and disconnected from the antenna.

Also, what other EMP weapons? There's the Ka'het (which might be an inspiration for the Remnant...), but I don't see any others. I don't see any balance issues with a non-purchasable thing like the Ka'het emp deployer have a similar disable damage. It only does 100 hull damage (compared to 700 shield damage), so having it drop to 60 disabled damage as well is not a huge drop.

Reply:
>  If the wire isn't even connected to the computer anymore because a chunk got blasted out of it earlier in the battle, then that wire isn't going to be conducting charge into the computer.

But the computer itself contains wires. It doesn't matter if it's connected to something else or not, it is the wires within the electronics that get overloaded and shorted out by an EMP.

> If you have a ham radio that is powered on and hooked up to an antenna, it will take a lot more EMP damage than an equivalent radio that is unpowered and disconnected from the antenna.

Are you sure? Because everything I see says that powering off electronics is no protection against an EMP.

Reply:
As best I can tell, it does. The powering off itself may not protect it; but disconnecting it from things that can pick up and channel more of the surge into it *can*. Also, circuits that are off can (and often will be, in the case of a ship that's taken a lot of damage) splintered into separate parts so that whereas the EMP hitting a wire in, say, the nose of the ship, won't be conducted to a computer in the engine room. 

As such, once the ship is disabled, each component only needs to deal with the EMP that is hitting it in specific, rather than being subject to all the conflicting waves of energy being transmitted through the wiring, which could effectivley lead to components being subject to energy spikes multiple times.

There's also the fact that some of that damage may have shorted and/or grounded out wiring, which further isolates components from additional damage. (proper grounding is one method of minimizing damage to electronics, too; but it is very hard to ground out everything while its running.)

Reply:
I think we're missing the point that hull damage is only done if the shields are down. If the argument being made is that the ionization overloads electronics and causes hull damage, then I would expect that hull damage to be dealt regardless of whether the shields are up or not, perhaps simply with a reduced impact against shielded targets just as the ion damage is shielded. That implies piercing, but I'd rather not tack even more onto weapons such as this. I would also expect all ion weapons to have this behavior, but they don't, and I don't think we should give that behavior to them.

Reply:
I will just point out that while EMPs may ionize, not all ionization is an EMP. These are two distinct effects that just happen to include some of the same effects, and, which I'm trying to move a little farther apart. The Teciimach is *not* just a really high power ion canon.

Reply:
To expand on that, I view ionization as a comparatively gentle charging of the whole ship to cause negative interactions with ship systems. EMP is an instantaneous burst intended to overload electrical circuitry, with the emphasis on being a hard "impact" to anything that relies on conductive systems. In this case, the ionization is more of an after-effect of the primary effect.